parser/symbol.go

```diff
@@ -79,5 +81,7 @@ func initUniverse() map[string]*symbol {
 		"false": {index: unsetAddr, value: vm.ValueOf(false), Type: vm.TypeOf(false)},
 
 		"println": {index: unsetAddr, value: vm.ValueOf(func(v ...any) { fmt.Println(v...) })},
+
+		"fmt": {kind: symPkg, index: unsetAddr, pkgPath: "fmt"},
```

Comment:
why is this needed here? does that mean that we have to add all packages here?

Reply:
The `fmt` symbol is not really needed here. The `import` statement and the fact that a matching binary package matches in `package.go` should be sufficient to proceed.

This is an experiment to pre-declare some package symbols (mainly from standard library) so they do not need to be imported explicetly and can be used directly. REPL mode in yaegi allows that, but it is not mandatory and can be achieved in a better way.

I will change that, thanks for the remark.